src/Flow.cpp

```diff
@@ -88,6 +88,22 @@ Flow::Flow(NetworkInterface *_iface,
       that complains, it means that the srv_host has made a connection that triggered
       the issue and thus it must be accounted in reverse
      */
+    u_int8_t type=icmp_info->getType();
+    u_int8_t code=icmp_info->getCode();
+
+    if(protocol==IPPROTO_ICMP){
+      if(type==3 && code==3){
```

Comment:
Do not hard-code constants. There are macros available for ICMP types and codes. Use them. Look in the ICMPinfo implementation for examples.